### DISCUSSION TITLE
My account does not log in

### DISCUSSION THREAD

**Discussion by crendonh** [CoffeaTeam/coffea-casa#413] [UNL tech support]
Hello everyone, I have a problem with my coffea.casa account, I am trying to log in but the server does not charge to enter to my account, It says: 
_____________________________________________________________________________
Your server is starting up.

You will be redirected automatically when it's ready for you.
_____________________________________________________________________________

And after few minutes .... -> Spawn failed: pod cmsaf-prod/jupyter-cesar-2erendon-2ehinestroza-40cern-2ech did not start in 300 seconds!

Help me please! thanks in advance.

I have attached the screenshot.
![coffea error](https://github.com/CoffeaTeam/coffea-casa/assets/71658925/ec31c3cf-83ce-46c5-8b8c-25744e5a8223)

**Comment by jthiltges** [comment]
Dear César,

After a recent power outage, a network filesystem mount did not recover on its own. It should now be working. Please give it a try now and let us know. 

Regards,
John

**Reply by crendonh** [reply]
Dear John,
Thanks so much! I have access now.

Kind regards,
César
